resources/user-agents/browsers/puffin/puffin-2-0.json

```diff
@@ -28,45 +28,81 @@
       },
       "children": [
         {
-          "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* (*khtml*like*gecko*) *Chrome/* Safari/* Puffin/#MAJORVER#.*AP Mobile",
+          "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* (*khtml*like*gecko*) *Chrome/* Safari/* Puffin/#MAJORVER#.*",
```

Comment:
The one on line 50 is just Android 3.2, 3.1 and 3.0 and is assigned the device "general Tablet".  The one here does not have 3.2, 3.1 and 3.0 android versions defined for that reason.